File: appstarter/src/main/java/org/jay/appstarter/utils/DispatcherLog.java
package org.jay.appstarter.utils;

import android.util.Log;

import org.jay.appstarter.BuildConfig;

// 日志打印器
public class DispatcherLog {

    private static boolean sDebug = BuildConfig.DEBUG;

    public static void i(String msg) {
        if (!sDebug) {
            return;
        }
        Log.i("TaskDispatcher",msg);
    }

    public static boolean isDebug() {
        return sDebug;
    }

    public static void setDebug(boolean debug) {
        sDebug = debug;
    }

}

File: appstarter/src/main/java/org/jay/appstarter/utils/DispatcherLog.kt
package org.jay.appstarter.utils

import android.util.Log
import org.jay.appstarter.BuildConfig
import org.jay.appstarter.utils.DispatcherLog

// 日志打印器
object DispatcherLog {

    var isDebug = BuildConfig.DEBUG

    @JvmStatic
    fun i(msg: String?) {
        if (!isDebug) {
            return
        }
        Log.i("TaskDispatcher", msg!!)
    }

}